vcpkg/get_started/overview.md

```diff
@@ -4,12 +4,93 @@ description: Learn about how the vcpkg C/C++ package manager can help you acquir
+
+## Why vcpkg?
+
+- Over 2200 open-source libraries to choose from, or bring your own
+- Consistent, cross-platform experience for Windows, macOS, and Linux
+- Dependencies can be built from source as needed with over 70 default build configurations (or customize your own builds)
+- Avoid ABI incompatibilities between packages and your consuming project
+- Prevent version conflicts and diamond problems in your dependency graph
+- Enterprise-ready tool relied by C/C++ developers across many industries, including: finance, gaming, embedded/IoT, semiconductors, defense, and at Microsoft
+- For MSBuild and CMake users: automatic integration with your build environment for a seamless dependency acquisition experience
+
+## vcpkg features
+
+### Ports and triplets
+
+A [port](../concepts/ports.md) in vcpkg is a versioned build recipe that produces a package. The most common type of package is a C/C++ library consisting of headers, source code, and binaries.
+
+A [triplet](../users/triplets.md) captures the target build environment (cpu, os, compiler, runtime, etc.) in a single, convenient name. vcpkg provides over 70 triplets by default, but you can also define your own.
+
+To install a package on your system, vcpkg runs the port's recipe file (a CMake script). The script may contain steps to download source code or run a build in your system. During builds, vcpkg uses the information in your triplet to make sure that the produced package matches your desired configuration.
+
+### Binary caching
+
+While vcpkg builds libraries from source whenever it's necessary, you can back up your built packages in a [binary cache](../consume/binary-caching-overview.md). This allows other developer machines or continuous integration runs to reference these prebuilt packages without running a new build every time. vcpkg intelligently determines if a rebuild is necessary by checking if the cache already contains a valid existing package when it attempts to install a package.
+
+### Manifests
+
+You can declare your direct dependencies and add optional features or version constraints in a [manifest file](../users/manifests.md).
+
+### Versioning
+
+vcpkg has a unique way of handling [package versions](../users/versioning.concepts.md). Your manifest file can reference a single, baseline version set by default. The baseline references a combination of ports, each at a specific version, that have been tested for compatibility against each other. Furthermore, you can set a minimum version for a package or override a package version to get an exact match.
```

Comment:
I think this paragraph could be reworded a bit to better emphasize "why" and "what" over "how". The purpose of baselines is to give you hassle-free, conflict-free dependency management with full reproducibility, while still giving you advanced control by pinning individual versions.